main.go

```diff
@@ -76,3 +77,27 @@ func main() {
 
 	os.Exit(0)
 }
+
+// TODO: This is unix system specific. When we get around to supporting windows we will want to
+// make this more platform agnostic.
+func getWrappedCommand() (cmdPath string) {
+	// Set this variable to the path of the cheque wrapper link
+	var chequePath = ""
+
+	var paths = strings.Split(os.Getenv("PATH"), ":")
+	for _, path := range paths {
+		cmdPath = fmt.Sprint(path, "/", context.GetCommand())
```

Comment:
Is the `context.GetCommand()` to distinguish between `gcc` and `glang` or something like that?

Reply:
It gives us the actual name we are called under. Our binary is "cheque", but if we are called through a "gcc" symlink then this command returns gcc instead of cheque.

In theory we could just use "os.Args[0]" directly, but I think the idea was that this abstracts it a bit if we need to do something else fancy in he future. For example if we override with a config value.